ERCS/erc-draft_contract-metadata.md

```diff
@@ -0,0 +1,136 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 8049
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.